kymatio/toolkit.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""Convenience utilities."""
```

Comment:
This collection of functions can probably find its place elsewhere in the codebase

Reply:
Sure. Where?